packages/client/src/features/change-bounds/movement-restrictor.ts

```diff
@@ -45,6 +45,8 @@ export function removeMovementRestrictionFeedback(element: SModelElement, moveme
 
 @injectable()
 export class NoOverlapMovmentRestrictor implements IMovementRestrictor {
+    public cssClasses = ["movement-not-allowed"];
```

Comment:
[Very,very minor]: Why did you opt for the explicit `public` modifier her? AFAIK we omit `public` in most other cases and should probably keep it consistent.

Reply:
Good point! I believe I added it explicitly as I had the feeling that we do add the `public` modifier more often than not, e.g., in all action constructors we use the public keyword. I am fine with both but it should be consistent. It is probably my Java background that tilts me a little bit more towards an explicit 'public' but I am also fine with omitting it. What do you think?

Reply:
Constructors are a different story, constructor variables are private by default, here you need the explicit `public`.
I think we should opt for implicit public were applicable. We are heavily reusing code from Theia and sprotty and both use implicit public as well. So I guess, we should do the same.

Reply:
Good point! I rebased the branch and removed the explicit modifiers in a separate commit. Thank you for the explanation!